zerver/lib/actions.py

```diff
@@ -6574,6 +6576,15 @@ def do_invite_users(
 ) -> None:
 
     check_invite_limit(user_profile.realm, len(invitee_emails))
+    if settings.BILLING_ENABLED:
+        try:
+            check_another_seat_can_be_added(user_profile.realm)
+        except LicenseLimitError:
+            error_message = _(
+                "Your organization is using all the licenses. Please increase the number of "
```

Comment:
This is when a new user is invited when an organization is already using all the licenses.

![Screenshot from 2021-05-25 16-42-09](https://user-images.githubusercontent.com/7190633/119488537-4bc72680-bd78-11eb-990b-2738716cf955.png)

Reply:
I think we need two different versions of this, depending on whether or not the user can add licenses or not.

If they can:
`All Zulip licenses for this organization are currently in use. To invite more users, please [increase the number of licenses](link) or [deactivate inactive users](link) to invite new users.`

If they can't: 
`You cannot invite new users because all Zulip licenses are in use. Please ask an administrator to [increase the number of licenses](link) or [deactivate inactive users](link), and try again.`